templates/default/sssd.conf.erb

```diff
@@ -4,7 +4,8 @@ services = nss, pam<% if node['sssd_ldap']['ldap_autofs'] %>, autofs<% end %><%
 domains = LDAP
 
 [nss]
-filter_users = root,named,avahi,haldaemon,dbus,radiusd,news,nscd
+filter_users = <%= node['sssd_ldap']['filter_users'].join(',') %>
+filter_groups = <%= node['sssd_ldap']['filter_groups'].join(',') %>
```

Comment:
I take that back.  It seems to depend on the type of argument the configuration directive takes.  For instance, an empty value for a configuration option that expects a string resolves to an empty string while an empty value for a configuration option that expects a boolean appears to be ignored. The `sssd.conf(5)` man page unfortunately is not clear on this issue.

With regards to `filter_groups = ` in particular, this is equivalent to `filter_groups = ""`, in other words, do not filter any groups.